.luacheckrc

```diff
@@ -0,0 +1,31 @@
+std = "lua51+minetest"
+unused_args = false
+allow_defined_top = true
+max_line_length = 999
+
+stds.minetest = {
+	read_globals = {
+		"DIR_DELIM",
```

Comment:
Undocumented. This should not be allowed explicitly.

Reply:
This one is taken from https://rubenwardy.com/minetest_modding_book/en/quality/luacheck.html#configuring-luacheck. We should also report this on https://gitlab.com/rubenwardy/minetest_modding_book.